IFComp/root/src/admin/index.tt

```diff
@@ -10,4 +10,8 @@
         <li><a href="[% c.uri_for('/admin/voting/') %]">Voting</a></li>
      </ul>
      
+     <ul>
+        <li><a href="[% c.uri_for('/admin/feedback/') %]">Feedback</a></li>
```

Comment:
If looks like the admin menu is now available to both votecounters and curators, but they can each access only one of the two available menu options. Let's show/hide the links based on whether the user has the appropriate role.